test/Driver/options.swift

```diff
@@ -4,9 +4,12 @@
 // NO_FILES: this mode requires at least one input file
 
 // RUN: not %target-swift-frontend -parse-sil -emit-sil 2>&1 | %FileCheck -check-prefix=SIL_FILES %s
-// RUN: not %target-swift-frontend -parse-sil -emit-sil %s %s 2>&1 | %FileCheck -check-prefix=SIL_FILES %s
```

Comment:
We've lost this test now (more than one input file). Maybe `%S/../Inputs/empty.swift` instead?

Reply:
I'll try it.